tasks/main.yml

```
@@ -422,6 +479,14 @@
     description: Microsoft SQL Server Tools
     baseurl: "{{ mssql_client_repository }}"
     gpgcheck: true
+  when: ansible_pkg_mgr in ["yum", "dnf"]
+
+- name: Configure the Microsoft SQL Server Tools repository # noqa command-instead-of-module
+  command: 'zypper addrepo -c -f --name "Microsoft SQL Server Tools" {{ mssql_client_repository }} packages-microsoft-com-prod'
```

```suggestion
  command: >-
    zypper addrepo -c -f --name "Microsoft SQL Server Tools" {{ mssql_client_repository }}
    packages-microsoft-com-prod
```
Quotes are not required, split a long line.

---

Thank you for removing dependency for community.general!